pkg/diagnose/cni.go

```diff
@@ -206,3 +222,97 @@ func mustHaveSubmariner(clusterInfo *cluster.Info) {
 		panic("cluster.Info.Submariner field cannot be nil")
 	}
 }
+
+func checkOVNVersion(info *cluster.Info, status reporter.Interface) bool {
+	status.Start("Checking OVN version")
+	defer status.End()
+
+	clientSet := info.ClientProducer.ForKubernetes()
+
+	ovnPod, err := findPod(clientSet, ovnKubeDBPodLabel)
+	if err != nil || ovnPod == nil {
+		status.Failure("Failed to get OVNKubeDB Pod %v", err)
+		return false
+	}
```

Comment:
`findPod` returns an error if not found so no need to check for nil `ovnPod`.

```suggestion
	if err != nil {
		status.Failure("Failed to get the OVN DB pod: %v", err)
		return false
	}
```